torch/csrc/distributed/rpc/process_group_agent.cpp

```diff
@@ -172,6 +174,10 @@ const WorkerInfo& ProcessGroupAgent::getWorkerInfo(worker_id_t id) const {
   return allWorkerInfo_[id];
 }
 
+const std::chrono::seconds& ProcessGroupAgent::getRpcTimeout() {
```

Comment:
We should store millisecond precision in this class and not at seconds precision.